assets/js/webauthn-registration.js

```diff
@@ -73,6 +73,15 @@ const registerPassKey = async () => {
   // POST the response to the endpoint that calls
   // @simplewebauthn/server -> verifyRegistrationResponse()
   registrationResponseStringInputElement.value = JSON.stringify(attResp);
+
+  const mfaProConnectConfig = document.getElementById("mfa-proconnect-config");
```

Comment:
todo: id should change into: "force_2fa". You will have to find a way to use a distinct selector.

Reply:
suggestion: 

```js
.querySelector('form[action="xxx"] input[name="force_2fa"]');
```